nodeup/pkg/model/context.go

```diff
@@ -62,6 +67,10 @@ type NodeupModelContext struct {
 
 	kubernetesVersion semver.Version
 	bootstrapCerts    map[string]*nodetasks.BootstrapCert
+
+	// ConfigurationMode determines if we are prewarming an instance or running it live
+	ConfigurationMode string
```

Comment:
Perhaps this should be its own type?

Reply:
I would have to think about where to put the type to avoid cyclic dependencies. That is why the const is not used close to your other comment.

Reply:
Can be done later.